### PR TITLE
Futures Prices Testnet

### PR DESCRIPTION
**Pull request by arturbeg**

### CODE REVIEW

File: packages/tomls/src/oracle_manager/configs/reyaMark/aave_usdc.toml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[invoke.oracle_manager_register_node_aave_usdc_mark_reya]
+target = ["OracleManagerProxy"]
+fromCall.func = "owner"
+func = "registerNode"
+args = [
+    "<%= settings.ORACLE_MANAGER_NODE_TYPE_REYA_OFFCHAIN_LOOKUP %>",
```
Comment (avniculae) [inline packages/tomls/src/oracle_manager/configs/reyaMark/aave_usdc.toml:6]: update constants.toml file

File: packages/tomls/src/oracle_manager/configs/storkMark/aave_usdc.toml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+target = ["OracleManagerProxy"]
+fromCall.func = "owner"
+func = "registerNode"
+args = [
+    "<%= settings.ORACLE_MANAGER_NODE_TYPE_DIV_REDUCER %>",
+    "0x",
+    "<%= defaultAbiCoder.encode(['address','string'], [contracts.OracleAdaptersProxy.address, settings.storkAssetPairIdAaveUSDCMark]) %>",
+    [
+        "<%= settings.aaveUsdMarkNodeIdStork %>",
+        "<%= settings.usdcUsdNodeIdStork %>",
```
Comment (avniculae) [inline packages/tomls/src/oracle_manager/configs/storkMark/aave_usdc.toml:11]: future prices do not apply to usdc-usd pair right?

File: packages/tomls/src/omnibus/reya_cronos.toml [hunk 3]
```diff
@@ -170,6 +170,19 @@ storkAssetPairIdCRV = "CRVUSD"
 storkAssetPairIdUNI = "UNIUSD"
```
Comment (avniculae) [inline packages/tomls/src/omnibus/reya_cronos.toml:170]: run `yarn lint:write` at the top of the project

File: packages/tomls/src/oracle_manager/configs/storkMark/aave_usdc.toml (hunk 2 above)
Comment (arturbeg) [reply packages/tomls/src/oracle_manager/configs/storkMark/aave_usdc.toml:11]: we normalise using the spot, following stork methodology

File: packages/tomls/src/oracle_manager/configs/reyaMark/aave_usdc.toml (hunk 1 above)
Comment (arturbeg) [reply packages/tomls/src/oracle_manager/configs/reyaMark/aave_usdc.toml:6]: https://github.com/Reya-Labs/reya-deployments/pull/124/commits/45e91d8d0c4c5db021596d6ba5b2f355d4b04192 ?

File: packages/tomls/src/oracle_manager/configs/reyaMark/op_usdc.toml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+target = ["OracleManagerProxy"]
+fromCall.func = "owner"
+func = "registerNode"
+args = [
+    "<%= settings.ORACLE_MANAGER_NODE_TYPE_STORK_OFFCHAIN_LOOKUP %>",
+    "<%= defaultAbiCoder.encode(['address','string'], [contracts.OracleAdaptersProxy.address, settings.reyaAssetPairIdOPMark]) %>",
+    [
+    ],
+]
+var.opUsdcNodeMarkIdReya.event = "NodeRegistered"
```
Comment (avniculae) [inline packages/tomls/src/oracle_manager/configs/reyaMark/op_usdc.toml:11]: this name looks wrong

Comment (arturbeg) [reply packages/tomls/src/oracle_manager/configs/reyaMark/op_usdc.toml]: fixed

File: packages/tomls/src/omnibus/reya_cronos.toml [hunk 5]
```diff
@@ -14,7 +14,7 @@ include = [
     "../orders_gateway/testnet.toml",
```
Comment (avniculae) [inline packages/tomls/src/omnibus/reya_cronos.toml:14]: need to replicate this on mainnet

Comment (avniculae) [reply packages/tomls/src/omnibus/reya_cronos.toml:14]: need to adjust tests

**Pull request merged**